ts/Dashboard/Component/Component.ts

```diff
@@ -584,8 +584,12 @@ abstract class Component<TEventObject extends Component.EventTypes = Component.E
     public render(): this {
         if (!this.hasLoaded) {
             this.load();
-            // Call resize to fit to the cell
-            this.resizeTo(this.parentElement);
+            // Call resize to fit to the cell. Timeout was added to make sure
+            // that each element is added to the DOM and that we have the
+            // correct dimensions.
+            setTimeout((): void => {
+                this.resizeTo(this.parentElement);
+            }, 100);
```

Comment:
If you are waiting for the DOM refresh cycle, a timeout of `0` is enough. If you need more than that, the timeout concept is not the best solution and `MutationObserver` should be used instead.

Reply:
Solved easier than expected.